salt/utils/verify.py

```diff
@@ -192,106 +189,164 @@ def verify_files(files, user):
     return True
 
 
-def verify_env(dirs, user, permissive=False, pki_dir='', skip_extra=False):
+def verify_env(dirs, user, default_mode=0o750,
+               allow_nonprimary_gid=False, pki_dir='', skip_extra=False):
```

Comment:
it is more of following python function upgrade semantics, the new args should be at the end of the function because it can be called without keywords passed in.
So I would not change the permissive arg here, and add `default_mode` to the end of the func